.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,16 @@
+Thank you for your contribution to OpenRA!
+
+Please be aware that we do not have enough project maintainers to match the rate of contributions, so it may take several days before somebody is able to respond to your Pull Request.
+
+You can help speed up the review process by following a few steps:
+
+* Make sure that you have read and understand the OpenRA Coding Standard (see https://github.com/OpenRA/OpenRA/wiki/Coding-Standard).
+* Write quality commit messages (see https://chris.beams.io/posts/git-commit/).
+* Only commit changes that directly relate to your Pull Request.  Use your Git interface to unstage any unrelated changes to project files, line endings, or other files.
+* Use the code diff view below to look for (and then fix!) whitespace errors in your code.
```

Comment:
The tooling should do this automatically. This should never be a manual process.

Reply:
Once again, this statement reflects the reality of the situation.  Many PRs from new contributors sit with failing tests until a reviewer points them out, and there are other cases (such as tabs vs spaces or project file changes) that are not tested by the automatic tooling.

Reply:
In that case I suggest promoting @EditorConfig and #14182 instead.